securitycenter/api/tests/SecurityCenterTests.cs

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using Xunit;
+
+[Collection(nameof(ConfigFixture))]
+public class SecurityCenterTests
+{
+    private readonly ConfigFixture _fixture;
+    public SecurityCenterTests(ConfigFixture fixture)
+    {
+        _fixture = fixture;
+    }
+
+    [Fact]
+    public void CreateNotificationConfig_ShouldCreateConfig()
+    {
+        string configId = "csharp-create-config-test111";
+
+        Assert.NotNull(CreateNotificationConfigSnippets.CreateNotificationConfig(_fixture.OrganizationId, configId, _fixture.ProjectId, _fixture.Topic));
+
+        DeleteNotificationConfigSnippets.DeleteNotificationConfig(_fixture.OrganizationId, configId);
+    }
+
+    [Fact]
+    public void DeleteNotificationConfig_ShouldDeleteConfig()
+    {
+        string configId = "csharp-delete-config-test111";
+        CreateNotificationConfigSnippets.CreateNotificationConfig(_fixture.OrganizationId, configId, _fixture.ProjectId, _fixture.Topic);
+
+        Assert.True(DeleteNotificationConfigSnippets.DeleteNotificationConfig(_fixture.OrganizationId, configId));
+    }
+
+    [Fact]
+    public void ListNotificationConfig_ShouldReturnNonEmpty()
+    {
+        string configId = "csharp-list-config-test111";
+        CreateNotificationConfigSnippets.CreateNotificationConfig(_fixture.OrganizationId, configId, _fixture.ProjectId, _fixture.Topic);
+
+        Assert.NotNull(ListNotificationConfigSnippets.ListNotificationConfigs(_fixture.OrganizationId));

```

Comment:
This will always pass unless the sample fails, because the value return will always be non-null, it might be an empty list (or empty IEnumerable). So I think that what you want to check is that this is not empty.